utils/src/command.rs

```diff
@@ -68,9 +69,20 @@ impl CommandRunExt for Command {
     fn run(&mut self) -> Result<()> {
         let stderr = tempfile::tempfile()?;
         self.stderr(stderr.try_clone()?);
+        tracing::trace!("exec: {self:?}");
         self.status()?.check_status(stderr)
     }
 
+    /// Output a debug-level log message with this command.
+    fn log_debug(&mut self) -> &mut Self {
+        // We unconditionally log at trace level, so avoid double logging
```

Comment:
To be pedantic, a user might able to avoid logging accidentally through running `output()`, `spawn()`, `status()`, etc instead of `run()`

Reply:
Yeah, I have no good ideas at this point for that other than basically "don't do that"?

I think effectively our extension trait's role is to take over those "terminal invocations"...it'd seem weird to use the trait just for the debug logging which is pretty trivial to do externally.

This would be the kind of thing I'd worry about if we decided to go make an official crate from our helpers, but...

Reply:
Yeah that's fair